tslearn/tests/test_metrics.py::test_masks fails

**Describe the bug**
tslearn/tests/test_metrics.py::test_masks fails

**To Reproduce**
```
git clone git@github.com:tslearn-team/tslearn.git
cd tslearn/
pip install -r requirements.txt
pip install pytest
pytest tslearn/tests/test_metrics.py
```

**Expected behavior**
All tests pass

**Actual Behavior**
```
>               np.testing.assert_allclose(
                    estimator1.fit(time_series).labels_, estimator2.fit(time_series).labels_
                )

tslearn/tests/test_metrics.py:436: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = (<function assert_allclose.<locals>.compare at 0x7fb66f7749d0>, array([0, 2, 1, 2, 2, 2, 1, 2, 0, 1, 2, 1, 0, 2, 2]), array([1, 0, 0, 1, 1, 0, 2, 1, 0, 2, 0, 2, 0, 1, 1]))
kwds = {'equal_nan': True, 'err_msg': '', 'header': 'Not equal to tolerance rtol=1e-07, atol=0', 'strict': False, ...}

    @wraps(func)
    def inner(*args, **kwds):
        with self._recreate_cm():
>           return func(*args, **kwds)
E           AssertionError: 
E           Not equal to tolerance rtol=1e-07, atol=0
E           
E           Mismatched elements: 13 / 15 (86.7%)
E           Max absolute difference among violations: 2
E           Max relative difference among violations: 1.
E            ACTUAL: array([0, 2, 1, 2, 2, 2, 1, 2, 0, 1, 2, 1, 0, 2, 2])
E            DESIRED: array([1, 0, 0, 1, 1, 0, 2, 1, 0, 2, 0, 2, 0, 1, 1])
```

**Environment (please complete the following information):**
 - OS:Ubuntu 20
 - Python 3.10
 - tslearn version lastest master

**Additional context**
N/A

--- a/tslearn/tests/test_metrics.py
+++ b/tslearn/tests/test_metrics.py
@@ -423,15 +423,20 @@ def test_masks():
                 metric_params={"itakura_max_slope": 1.0},
                 max_iter=5,
                 random_state=0,
+                init="random"
             )
             estimator2 = tslearn.clustering.TimeSeriesKMeans(
-                metric="euclidean", max_iter=5, random_state=0
+                metric="euclidean",
+                max_iter=5,
+                random_state=0,
+                init="random"
             )
             estimator3 = tslearn.clustering.TimeSeriesKMeans(
                 metric="dtw",
                 metric_params={"sakoe_chiba_radius": 0},
                 max_iter=5,
                 random_state=0,
+                init="random"
             )
             np.testing.assert_allclose(
                 estimator1.fit(time_series).labels_, estimator2.fit(time_series).labels_

